Applications/ApplicationsLib/UncoupledProcessesTimeLoop.h

```diff
@@ -199,18 +204,22 @@ template<typename Matrix, typename Vector>
 std::unique_ptr<UncoupledProcessesTimeLoop<Matrix, Vector> >
 createUncoupledProcessesTimeLoop(BaseLib::ConfigTree const& conf)
 {
-    auto const type = conf.getConfParam<std::string>("type");
+    auto const type = conf.peekConfParam<std::string>("type");
 
-    if (type == "SingleStep")
-    {
-        using TimeLoop = UncoupledProcessesTimeLoop<Matrix, Vector>;
-        return std::unique_ptr<TimeLoop>(new TimeLoop);
-    }
-    else
-    {
+    std::unique_ptr<NumLib::ITimeStepAlgorithm> timestepper;
+
+    if (type == "SingleStep") {
```

Comment:
But "SingleStep" is more explicit than having to configure `FixiedTimeStepping` with suitable t0, t_end, delta_t s.t. only exactly one timestep is computed.